lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -1960,9 +1960,7 @@ struct NodeBypass : public mlir::RewritePattern {
     if (node.getInnerSym() || !AnnotationSet(node).canBeDeleted() ||
         node.use_empty() || node.isForceable())
       return failure();
-    rewriter.startOpModification(node);
-    node.getResult().replaceAllUsesWith(node.getInput());
```

Comment:
This modifies operations that aren't `node` (unless input is its own result), change to use RAUW via rewriter.